examples/custom-resources/jwks/README.md

```diff
@@ -0,0 +1,157 @@
+# JWKS
+
+In this example we deploy a web application, configure load balancing for it via a VirtualServer, and apply a JWT policy.
+Instead of using a local secret to verify the client request like we do in our [jwt](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/jwt) example, we will define an external Identity Provider (IdP) using the `JwksURI` field.
```

Comment:
```suggestion
Instead of using a local secret to verify the client request such as in the [jwt](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/jwt) example, we will define an external Identity Provider (IdP) using the `JwksURI` field.
```